text/0000-channel-participation-api-without-system-channel.md

```diff
@@ -0,0 +1,497 @@
+[guide-level-explanation]: #guide-level-explanation
+
+We introduce a new mode of channel participation management that does not depend on the existence of a system channel.
+
+We introduce the ability to start an OSN process without any channels. The OSN exposes a new **"channel participation" 
+API** that allows the operator to join and leave channels on that specific OSN.   
+
+A **"local orderer channel participation administrator"** is a role that is associated with the operation and 
+administration of an OSN. The identity associated with this role is able to invoke a new **"channel participation" 
+API**, and perform the following commands:
+
+- **Join** this OSN to a new or an existing channel,
+- **Remove** this OSN from an existing channel,
+- **List** the channels this OSN is a member of.   
+
+The new mode of channel participation management is meant to be incompatible with channel creation using a system 
+channel, in the sense that if the system channel exists, the `Join` operation will not be supported. 
+However, there is a transition path from a network operated with a system channel to a network that is operated without
+one. In addition, it should be possible to start OSN processes without channels, create a system channel with the
+channel participation API, and continue to operate the network in the usual way - creating channels through the system 
+channel. "Mixed mode" management, that is, using the system channel to create channels on some OSNs and the 
+participation API on other OSNs is not supported and highly discouraged.  
+
+## Channel participation commands 
+In this section channel participation commands are described in a high level. The commands are presented in REST style
+but some details (i.e. exact formats, responses) are omitted for clarity.
+   
+### Joining a channel
+A channel is joined by providing the OSN with a config block:
+
+  ```Join channel-name channel-config-block```
+  
+  or
+  
+  `POST /channels/channel-name` with body containing `channel-config-block`
+  
+The body contains a config block that contains all the details about the channel. 
+
+If the OSN is joining a new channel, the config block is the genesis block of the channel, generated by the 
+`configtxgen` tool, as described below. It looks like the first block of an 
+application channel that is generated by the OSN, when a channel creation transaction is submitted to the system channel.
+
+If the OSN is joining an existing channel, the config block is simply the last config block of the channel, retrieved 
+from one of the other OSNs.
+
+If the channel already exists, the call will fail.
+
+If the system channel exists, the call will fail. 
+          
+The OSN receiving this call will inspect the block and conduct some sanity checks in order to make sure that the block 
+is well-formed.
+
+The OSN will try to find itself in the consenters set, and determine if it is a follower or a member of the cluster
+executing consensus. 
```

Comment:
The roles of "follower" and "member" should be more formally defined. It's referenced here with assumptions about the reader's definition of those words which may not match the intent.

Reply:
Agreed. I will improve the presentation.